blurb/blurb.py

```diff
@@ -1596,6 +1596,10 @@ def handle_option(s, dict):
         print()
         print("usage: ", end="")
         help(subcommand)
+    except subprocess.SubprocessError as e:
+        output = e.stdout.decode() + e.stderr.decode()
+        print("Failed running {}, output: \n{}".format(e.cmd, output))
```

Comment:
e.cmd is printed out by the exception--see the last line of the sample output you included in your PR.

Reply:
Should we change it to just "Failed running command"? as we are reraising?

You can see the sample output in the PR description